auto_cpufreq/bin/auto_cpufreq.py

```diff
@@ -137,59 +110,32 @@ def config_info_dialog():
                     print()
                     break
             conf.notifier.stop()
-        elif stats:
-            not_running_daemon_check()
+        elif daemon:
             config_info_dialog()
-            print('\nNote: You can quit stats mode by pressing "ctrl+c"')
-            if os.getenv("PKG_MARKER") == "SNAP":
+            root_check()
+            file_stats()
+            if IS_INSTALLED_WITH_SNAP and dcheck == "enabled":
                 gnome_power_detect_snap()
                 tlp_service_detect_snap()
-            else:
+            elif not IS_INSTALLED_WITH_SNAP:
                 gnome_power_detect()
                 tlp_service_detect()
-            battery_get_thresholds()
-            read_stats()
-        elif log: deprecated_log_msg()
-        elif get_state:
-            not_running_daemon_check()
-            override = get_override()
-            print(override)
-        elif debug:
-            # ToDo: add status of GNOME Power Profile service status
-            config_info_dialog()
-            root_check()
-            battery_get_thresholds()
-            cpufreqctl()
-            footer()
-            distro_info()
-            sysinfo()
-            print()
-            app_version()
-            print()
-            python_info()
-            print()
-            device_info()
-            print(f"Battery is: {'' if charging() else 'dis'}charging")
-            print()
-            app_res_use()
-            display_load()
-            get_current_gov()
-            get_turbo()
-            footer()
-        elif version:
-            footer()
-            distro_info()
-            app_version()
-            footer()
-        elif donate:
-            footer()
-            print("If auto-cpufreq helped you out and you find it useful ...\n")
-            print("Show your appreciation by donating!")
-            print("https://github.com/AdnanHodzic/auto-cpufreq/#donate")
-            footer()
+            battery_setup()
+            conf.notifier.start()
+            while True:
```

Comment:
Were all the changes on the left moved to some other lines? 

As I see big debug part being moved to i.e start 233:

```
        elif debug:
            # ToDo: add status of GNOME Power Profile service status
            config_info_dialog()
            root_check()
            battery_get_thresholds()
            cpufreqctl()
            footer()
            distro_info()
            sysinfo()
            print()
            app_version()
            print()
            python_info()
            print()
            device_info()
            print(f"Battery is: {'' if charging() else 'dis'}charging")
            print()
            app_res_use()
            get_load()
            get_current_gov()
            get_turbo()
            footer()
```

Reply:
The order has been changed to match the order of commands when you do `auto-cpufreq --help`

Reply:
Thought, that was the case, thanks for the clarification.